tests/integration/630.serving-functions-go.test.js

```diff
@@ -112,3 +112,72 @@ test('Updates a Go function when a file is modified', async (t) => {
     }
   })
 })
+
+// Reproduction test to verify the abscence/presence of a Go scheduled function
+test('Detects a Go scheduled function using netlify-toml config', async (t) => {
+  const [execaMock, removeExecaMock] = await createExecaMock(`
+    const { writeFileSync } = require('fs')
+
+    const handler = (...args) => {
```

Comment:
@karagulamos I wonder if we could assert that the function receives `nextRun` in the body?

Reply:
Not a blocker though, so please feel free to do that on a separate PR if you prefer.

Reply:
Oh, it's resolved.